subnet/subnet_test.go

```diff
@@ -41,7 +41,7 @@ func newDummyRegistry() *MockSubnetRegistry {
 		{ip.IP4Net{ip.MustParseIP4("10.3.5.0"), 24}, attrs, exp, 13},
 	}
 
-	config := `{ "Network": "10.3.0.0/16", "SubnetMin": "10.3.1.0", "SubnetMax": "10.3.5.0" }`
+	config := `{ "Network": "10.3.0.0/16", "SubnetMin": "10.3.1.0", "SubnetMax": "10.3.25.0" }`
```

Comment:
It used to check that AcquireLease worked by making sure it go `10.3.3.0/24` subnet because it configured a very narrow range and there was only one valid subnet left to allocate. This doesn't work with reservations b/c it may allocate out of SubnetMin/SubnetMax range if there's a reservation there. I now check via `inAllocatableRange` and so gave "room to breathe" -- this way it can do more than one allocation, if needed.